mtags/src/main/scala-3/scala/meta/internal/pc/CompilerInterfaces.scala

```diff
@@ -31,8 +31,9 @@ object CompilerInterfaces {
     val sourceFile = toSource(uri, text)
     // we need to have a separate reporter otherwise errors are not cleared
     val parsingReporter = new StoreReporter(null)
-    val freshContext = driver.currentCtx.fresh.setReporter(parsingReporter)
-    val parser = new Parser(sourceFile)(freshContext)
+    implicit val freshContext =
+      driver.currentCtx.fresh.setReporter(parsingReporter)
+    val parser = new Parser(sourceFile)
```

Comment:
Since we are switching this maybe a good idea to change over to Dotty syntax?
```suggestion
	import dotty.tools.dotc.core.Contexts.FreshContext
	
    given fc as FreshContext =
      driver.currentCtx.fresh.setReporter(parsingReporter)
    val parser = new Parser(sourceFile)
```

Reply:
Added using here.